share/src/main/java/org/alfresco/web/site/servlet/config/AIMSConfig.java

```diff
@@ -44,18 +45,51 @@ public void init()
     public boolean isAIMSEnabled()
     {
         Config AIMSConfigCondition = this.configService.getConfig(AIMSConfigElement.AIMS_CONFIG_CONDITION);
-        AIMSConfigElement config = null;
+        AIMSConfigElement aimsConfigElement = null;
 
         if (AIMSConfigCondition != null)
         {
-            config = (AIMSConfigElement) AIMSConfigCondition.getConfigElement(AIMSConfigElement.AIMS_CONFIG_ELEMENT);
+            ConfigElement config = (ConfigElement) AIMSConfigCondition.getConfigElement(AIMSConfigElement.AIMS_CONFIG_ELEMENT);
```

Comment:
I think AIMSConfigCondition.getConfigElement already returns a ConfigElement. I don't see the point for casting.